pkg/configuration/load_test.go

```diff
@@ -236,3 +235,68 @@ func createConfigMap(name, namespace string, data map[string]string) *v1.ConfigM
 		Data: data,
 	}
 }
+
+func TestGetWatchNamespaceWhenSet(t *testing.T) {
+	// given
+	restore := test.SetEnvVarAndRestore(t, "WATCH_NAMESPACE", "member-operator")
```

Comment:
Should we add a test for when it's set to empty `""`? Similar to OPERATOR_NAME below?

Reply:
This is a very good point and it also brings an important question - shouldn't it fail when it is set to an empty string...? Since we want to use namespace-scoped operators, then my guess is that yes - it should fail.
Fixed and improved here: 7979364

Reply:
Agreed, I think it should fail when empty as well as well.